tests/c_languages/ClangBearTest.py

```diff
@@ -1,4 +1,3 @@
-
 from bears.c_languages.ClangBear import ClangBear
```

Comment:
This as well ;)

Reply:
@newbazz as I told you before :) this patch was shown in CI just because of feature that I have added ;)
you can see I deleted leading blanklines from these files.